script.js

```diff
@@ -62,14 +80,48 @@ function addKeys() {
 
 function addNotes() {
 	let notes = document.createElement("p");
-	notes.innerHTML = "Windows OS";
+	notes.innerHTML = "Windows OS<br>Language change: Shift + Alt";
 	wrapper.appendChild(notes);
 }
 
 document.addEventListener("keydown", function (event) {
-	document.getElementById(event.code).classList.add("key-active");
+
+	if (document.getElementById(event.code)) {
+		document.getElementById(event.code).classList.add("key-active");
+		keysPressed.push(event.code);
+		if ((keysPressed[0] == "ShiftLeft" && event.code == "AltLeft") ||
+			(keysPressed[0] == "AltLeft" && event.code == "ShiftLeft")) {
+			changeLang();
+		}
+	}
 });
 
 document.addEventListener("keyup", function (event) {
-	document.getElementById(event.code).classList.remove("key-active");
+
+	if (document.getElementById(event.code)) {
+		document.getElementById(event.code).classList.remove("key-active");
+		keysPressed.length = 0;
+	}
 });
+
+function changeLang() {
+	lang == "en" ? lang = "ru" : lang = "en";
+	changeKeys();
+	localStorage.setItem("lang", lang);
+}
+
+function changeKeys() {
+	let alphabet;
+	let keyboardKeys = document.getElementsByClassName("keyboard")[0];
+
+	if (lang == "en") {
+		alphabet = keys;
+	} else {
+		alphabet = russianKeys;
+	}

```

Comment:
If you will use earlier mentioned approach (one of the first comments), you can replace all this lines by 
``` javascript
const alphabet = virtualKeyboardsKeys[lang];
```